.eslintignore

```diff
@@ -9,69 +9,27 @@ packages/cli/src/lib/live-server/*
-packages/core/src/html/includePanelProcessor.js
-packages/core/src/html/NodeProcessor.js
-packages/core/src/html/scriptAndStyleTagProcessor.js
-packages/core/src/html/SiteLinkManager.js
-packages/core/src/lib/nunjucks-extensions/index.js
-packages/core/src/lib/nunjucks-extensions/nunjucks-date.js
-packages/core/src/lib/nunjucks-extensions/set-external.js
-packages/core/src/Page/PageSources.js
-packages/core/src/plugins/codeBlockButtonsAssets/codeBlockButtonsContainer.js
-packages/core/src/plugins/codeBlockCopyButtons.js
-packages/core/src/plugins/codeBlockWrapButtons.js
-packages/core/src/plugins/types.js
-packages/core/src/variables/VariableProcessor.js
-packages/core/src/variables/VariableRenderer.js
-packages/core/src/Page/PageVueServerRenderer.js
-packages/core/src/External/External.js
-packages/core/src/External/ExternalManager.js
-packages/core/src/html/vueSlotSyntaxProcessor.js
-packages/core/src/html/MdAttributeRenderer.js
-packages/core/src/html/MarkdownProcessor.js
-packages/core/src/html/warnings.js
-packages/core/src/plugins/Plugin.js
-packages/core/src/plugins/PluginManager.js
-packages/core/src/html/FootnoteProcessor.js
-packages/core/src/html/headerProcessor.js
-packages/core/src/html/codeblockProcessor.js
-packages/core/src/html/siteAndPageNavProcessor.js
-packages/core/src/Page/index.js
+packages/core/src/errors/**/*.js
+packages/core/src/External/*.js
+packages/core/src/html/**/*.js
+packages/core/src/Layout/*.js
+packages/core/src/lib/nunjucks-extensions/*.js
 packages/core/src/lib/markdown-it/index.js
-packages/core/src/lib/markdown-it/utils/index.js
-packages/core/src/lib/markdown-it/highlight/helper.js
-packages/core/src/lib/markdown-it/highlight/HighlightRule.js
-packages/core/src/lib/markdown-it/highlight/HighlightRuleComponent.js
-packages/core/src/lib/markdown-it/plugins/markdown-it-icons.js
-packages/core/test/unit/html/NodeProcessor.test.js
-packages/core/test/unit/html/NodeProcessor.data.js
-packages/core/test/unit/html/linkProcessor.test.js
-packages/core/test/unit/html/includePanelProcessor.test.js
-packages/core/test/unit/html/SiteLinkManager.test.js
-packages/core/test/unit/utils/utils.js
-packages/core/test/unit/utils/data.js
-packages/core/src/plugins/default/markbind-plugin-anchors.js
-packages/core/src/plugins/default/markbind-plugin-plantuml.js
-packages/core/src/plugins/default/markbind-plugin-shorthandSyntax.js
-packages/core/src/plugins/default/markbind-plugin-tree.js
-packages/core/test/unit/plugins/default/anchor.test.js
-packages/core/test/unit/plugins/default/plantuml.test.js
-packages/core/test/unit/plugins/default/shorthandSyntax.test.js
-packages/core/test/unit/plugins/default/tree.test.js
-packages/core/src/plugins/algolia.js
-packages/core/src/plugins/disqus.js
-packages/core/src/plugins/filterTags.js
-packages/core/src/plugins/googleAnalytics.js
-packages/core/src/plugins/mathDelimiters.js
+packages/core/src/lib/markdown-it/highlight/*.js
+packages/core/src/lib/markdown-it/utils/*.js
```

Comment:
I think the packages/core/src/lib/markdown-it/plugins/markdown-it-icons.js needs to be added also! :)

Reply:
There's a section on the markdown-it plugins below (around lines 34-37), would that give the desired behaviour of `.eslintignore` ?

Reply:
Oh yes you are right!